### PR TITLE
Feature/sembench container (includes PR #19)

### PR DESCRIPTION
**Pull request by bulricht**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -97,6 +97,23 @@ services:
...
+    build:
+      context: ./docker/lwua-sembench/
+    image: lwua/lwua_sembench
+    container_name: lwua_sembench
+    volumes:
+      - ./data:/data
+      - ./config:/config
+    environment:
+      - INPUT_DATA_LOCATION=/data/input-data
+      - OUTPUT_DATA_LOCATION=/data/output-data
```
Comment (marc-portier) [inline docker-compose.yml]: think input and data will just sit together throughout the /data folder 
so both to just point to `/data` 

also best to indicate what this actually does --> I assume the paths in config/sembench.json can refer to these, or are assumed relative to these?

File: docker-compose.yml [hunk 2]
```diff
@@ -97,6 +97,23 @@ services:
...
+      - SEMBENCH_DATA_LOCATION=/data/sembench-data
```
Comment (marc-portier) [inline docker-compose.yml:116]: unclear role?

also: no path needed for templates?

File: config/sembench.json
Comment (marc-portier) [inline config/sembench.json:1]: too bad json is not comments-friendly

best to have some docs or example on how to use this

**Review by marc-portier — CHANGES_REQUESTED** [review]
best to update thos .env paths maybe?
otherwise looks good to go
maybe consider to have some docs on how to use it (or ref to those to be found online) 
would also welcome some way of automated testing

File: docker-compose.yml (hunk 2 above)
Comment (bulricht) [reply docker-compose.yml:116]: `sembench-data` is the folder that contains all data required to run pysembench. At the moment the templates should go in there, but e.g. shacl files as well

File: docker-compose.yml (hunk 1 above)
Comment (bulricht) [reply docker-compose.yml]: those paths are considered as a base path in the sembench.json yes

File: config/sembench.json
Comment (bulricht) [reply config/sembench.json:1]: pysembench docs should be (or should become) sufficient to construct this config file. I'm thinking now to switch to json5 or yaml to make it more comments-friendly

**Review by cedricdcc — APPROVED** [review]

**Pull request merged**